server/src/main/java/org/elasticsearch/search/aggregations/metrics/GeoBoundsAggregator.java

```diff
@@ -44,12 +45,9 @@
 
     private final ValuesSource.GeoPoint valuesSource;
     private final boolean wrapLongitude;
-    DoubleArray tops;
-    DoubleArray bottoms;
-    DoubleArray posLefts;
-    DoubleArray posRights;
-    DoubleArray negLefts;
-    DoubleArray negRights;
+    private ObjectArray<GeoExtent> extents;
+    // size in bytes of an extent object for accounting
+    private static int EXTENT_WEIGHT = 6 * Double.BYTES;
```

Comment:
I wonder if this should be a property of `GeoExtent` instead?  Or a static method which returns the size?  Just seems like it should live with the object, in case it every changes in the future we won't have to hunt for all the places that use the internal size.